source/packages/libraries/core/deployment-helper/infrastructure/cfn-deployment-helper-vpc.yaml

```diff
@@ -41,7 +41,7 @@ Resources:
       FunctionName: !Sub "cdf-cfnCustomResourcesVPC-${Environment}"
       CodeUri: ../bundle.zip
       Handler: lambda_custom_resource_proxy.handler
-      MemorySize: 128
```

Comment:
for unclear reasons, the DeploymentHelper Lambdas were having out-of-memory errors with this change. There's no downside to bumping up these memory limits, so I'm making that change.

Reply:
Unpacked size of the entire v2 AWS SDK is 85 megs: https://www.npmjs.com/package/aws-sdk